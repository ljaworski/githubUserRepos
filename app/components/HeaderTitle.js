import React from "react";
import PropTypes from "prop-types";
import { Text } from "native-base";

export default function HeaderTitle({ title }) {
  return <Text style={{ color: "#fff" }}>{title}</Text>;
}

HeaderTitle.propTypes = {
  title: PropTypes.string
};
