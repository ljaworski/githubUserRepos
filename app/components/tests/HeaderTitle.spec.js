import React from "react";
import HeaderTitle from "../HeaderTitle";
import renderer from "react-test-renderer";

const props = {
  title: "Test"
};

test("renders correctly", () => {
  const headerTitle = renderer.create(<HeaderTitle {...props} />).toJSON();
  expect(headerTitle).toMatchSnapshot();
});
