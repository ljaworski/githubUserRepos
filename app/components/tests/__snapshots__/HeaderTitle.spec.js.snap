// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#000",
        "fontFamily": "System",
        "fontSize": 16,
      },
      Object {
        "color": "#fff",
      },
    ]
  }
  uppercase={false}
>
  Test
</Text>
`;
